recipes/spz/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set version = "v2.0.0" %}
+
+package:
+  name: spz
+  version: {{ version }}
+
+source:
+  url: https://github.com/nianticlabs/spz/archive/{{ version }}.tar.gz
+  sha256: fa14626b531c1dee2c29c29b43de8eaa8b773c3b0ef434ee8372c9712dfb6773
+  patches:
+    - ./patches/0001-Add-BUILD_SHARED_LIBS-option.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('spz', max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - ninja
+  host:
+    - zlib
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/load-spz.h  # [unix]
+    - test -f ${PREFIX}/lib/libspz${SHLIB_EXT}  # [unix]
+    - test ! -f ${PREFIX}/lib/libspz.a          # [unix]
+    - if not exist %LIBRARY_BIN%\\spz.dll exit 1  # [win]
```

```suggestion
    - if not exist %LIBRARY_BIN%\spz.dll exit 1  # [win]
    - if not exist %LIBRARY_LIB%\spz.lib exit 1  # [win]
```
We don't need to test for the static library since we aren't building it.  Instead we also need to check for the `.lib` file on Windows.